projects/plugins/jetpack/extensions/blocks/like/like.php

```diff
@@ -72,6 +67,19 @@ function render_block( $attr, $content, $block ) {
 	 */
 	$new_layout = apply_filters( 'likes_new_layout', true ) ? '&amp;n=1' : '';
 
+	static $master_iframe_added = false;
```

Comment:
Can we call this `$primary_iframe_added`? ([context](https://www.wired.com/story/tech-confronts-use-labels-master-slave/))

Reply:
I've updated the PR.

I've another commit ready to push that replaces the name of the function `jetpack_likes_master_iframe` to `jetpack_likes_main_iframe` and updates all references. Is it ok to do this in one go? Or is it too risky, as we might break other plugins that are somehow relying on this function to be there? (*)

(* we could stub it)

Reply:
I'd do that separately and deprecate the function so it doesn't break things